manual/source/nxdl-types.rst

```diff
@@ -12,7 +12,7 @@ Field Types allowed in NXDL specifications
 ##########################################
 
 Field types for use in NXDL
```

Comment:
I guess `Field types` should be called `Data types` then.

Reply:
Done